script.js

```diff
@@ -0,0 +1,108 @@
+const todoInput = document.querySelector("#todo-input");
+const addButton = document.querySelector("#add-button");
+const todoForm = document.querySelector("#enter-todo");
+const todoList = document.querySelector("#todo-list");
+const doneList = document.querySelector("#done-list");
+
+let todoStore = [];
+let doneStore = [];
+
+todoForm.addEventListener("submit", function (event) {
+  event.preventDefault();
+  addTodo();
+});
+
+function saveTodos() {
+  localStorage.setItem("todoStore", JSON.stringify(todoStore));
+  localStorage.setItem("doneStore", JSON.stringify(doneStore));
+}
+
+function deleteTodo(event) {
+    event.stopPropagation();
+    const todoItem = event.target.parentNode;
+    const todoId = todoItem.id;
+  
+    if (todoItem.parentNode.id === "todo-list") {
+      const index = todoStore.findIndex((todo) => todo.id === todoId);
+      if (index !== -1) {
+        todoStore.splice(index, 1);
+      }
+    } else {
+      const index = doneStore.findIndex((done) => done.id === todoId);
+      if (index !== -1) {
+        doneStore.splice(index, 1);
+      }
+    }
+    todoItem.remove();
+    saveTodos();
+  }
```

Comment:
여기도 기능을 조금 분리하면 가독성이 더 좋을 것 같아요!!
```suggestion
function deleteTodo(event) {
  event.stopPropagation();
  const todoItem = event.target.parentNode;
  const todoId = todoItem.id;
  const isTodoList = todoItem.parentNode.id === "todo-list";

  if (isTodoList) {
    deleteItem(todoId, todoStore);
  } else {
    deleteItem(todoId, doneStore);
  }

  todoItem.remove();
  saveTodos();
}

function deleteItem(itemId, store) {
  const index = store.findIndex((item) => item.id === itemId);
  if (index !== -1) {
    store.splice(index, 1);
  }
}

```